src/services/helpers/types.ts

```diff
@@ -62,4 +62,6 @@ export enum WorkspaceAction {
   DELETE_WORKSPACE = 'Delete Workspace',
   ADD_PROJECT = 'Add Project',
   ADD_CUSTOM_WORKSPACE = 'Add Workspace',
+  RESTART_WORKSPACE = 'Restart Workspace',
```

Comment:
Start in backgroud in my case opens loading page, pay attention to error:
![Screenshot_20210402_094011](https://user-images.githubusercontent.com/5887312/113388769-7c519c00-9397-11eb-8604-532a0ca69d45.png)